This entware patch to fix dynamic loader. It can be combined with 0001 openwrt patch, but we keep it separate
--- a/make-linux.mk
+++ b/make-linux.mk
@@ -70,7 +70,7 @@
 	override CFLAGS+=-Wall -Wno-deprecated -pthread $(INCLUDES) -DNDEBUG $(DEFS)
 	CXXFLAGS?=-O3 -fstack-protector -fPIE
 	override CXXFLAGS+=-Wall -Wno-deprecated -std=c++11 -pthread $(INCLUDES) -DNDEBUG $(DEFS)
-	LDFLAGS=-pie -Wl,-z,relro,-z,now
+	LDFLAGS+=-pie -Wl,-z,relro,-z,now
 	RUSTFLAGS=--release
 endif
 
